arm-recoveryservices/2016-06-01/swagger/vaultusages.json

```diff
@@ -0,0 +1,188 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/usages": {
+      "get": {
+        "tags": [
+          "VaultUsages"
+        ],
+        "description": "Fetches the usages of the vault.",
+        "operationId": "VaultUsages_List",

```

Comment:
Vault usage represents usage per vault. The guideline states that the first half of Operation ID should be noun; VaultUsage is a compound noun. What is the issue with this name?